### DISCUSSION TITLE
Is there a way to using match with different login names?

### DISCUSSION THREAD

**Discussion by tomvisions** [fastlane/fastlane#17759] [General]
Hello, Trying to use fastlane match development to setup all the certs. Unfortunately, my login for the apple developer portal is different then the one for git. When running the command it mentions the git user needs to be the same as the apple developer user. Can this be overridden?

**Comment by rogerluan** [comment]
Hey @tomvisions 👋 

>it mentions the git user needs to be the same as the apple developer user. 

Can you point me to where it's saying that? For instance copying and pasting the exact message here?
That statement doesn't seem to be correct, as I use match with a GitHub account that has a different email than the Apple developer account I'm working with.

Waiting for more info 🙏
